src/test/java/Homework16.java

```diff
@@ -0,0 +1,59 @@
+import org.checkerframework.checker.units.qual.C;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 {
+
+    @Test
+    public static void createNewPlaylist () throws InterruptedException {
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+        String url = "https://bbb.testpro.io";
+        driver.get(url);
+
+        WebElement emailLogin = driver.findElement(By.cssSelector("[type='email']"));
+        emailLogin.click();
+        emailLogin.sendKeys("tasiakwiggins@gmail.com");
+
+        WebElement passwordBox = driver.findElement(By.cssSelector("[type='password']"));
+        passwordBox.click();
+        passwordBox.sendKeys("te$t$tudent");
+
+        WebElement submitButton = driver.findElement(By.cssSelector("[type='submit']"));
+        submitButton.click();
+        Thread.sleep(5000);
+
+        WebElement createPlaylist = driver.findElement(By.xpath("//*[@title='Create a new playlist']"));
```

Comment:
you can use driver.findElement(By.xpath("//*[@title='Create a new playlist']")).click();